src/content/docs/alerts-applied-intelligence/notifications/notification-integrations.mdx

```diff
@@ -884,6 +888,28 @@ If you're building a new service to consume this payload you must test it locall
 
 For notifications sent from workflows, you can acknowledge or close issues using [Nerdgraph](/docs/apis/nerdgraph/examples/nerdgraph-issues-api-via-github/#resolve-ack-issues). When testing two-way sync with webhooks, you can use Beeceptor's customized response status and payload templates. This allows you to draft a desired response when acknowledging the received events.
 
+### Secure URL [#secure-url-webhook]
+
+You can add secrets in the URL’s path and parameters, such as service ID or API keys. To avoid having these secrets exposed a masking option has been added. This option will mask everything past the domain, masking it from users and internal metrics alike. The full URL will only be used upon sending notifications. Due to the nature of this sensitive data, once the URL has been saved, the only way to edit it will be to completely override the previous URL.
+
+  <img
+      width="80%;"
+      title="Webhook destination secure URL"
+      alt="Webhook destination secure URL"
+      src={accountsWebhookSecureUrl}
+    />
+
+### Custom Headers Authentication [#custom-headers-auth-webhook]
+
+Many users need to pass certain secrets in their headers, such as API keys and personal IDs. To prevent this sensitive information from being exposed, the custom headers authentication has been created. This will let you create headers that will be added to your notifications, while keeping their values encrypted and masked from both users and internal logs alike. In case a change in the value, the complete deletion and re-writing of the value will be needed. In case the same headers key will be used in both the Workflow channel and destination, the destination header will take precedence, and will be used instead of the channel header. 
```

Comment:
```suggestion
In case the same headers key will be used in both the Workflow channel and destination, the destination header will take precedence, and will be used instead of the channel header. 
```

hi @nzur-newrelic ! Thanks for adding this content. Can you unspool this copy a little more?

- what are workflow channels?
- What are destinations in terms of header?
- How does a header key relate to the value assigned to sensitive information? 
- What is the "destination header taking precedence" over? 
- What are the differences between destination and channel headers? How do values relate to them?